x-pack/plugins/uptime/public/routes.test.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// app.test.js
+import { screen } from '@testing-library/react';
+import { render } from './lib/helper/rtl_helpers';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import * as telemetry from './hooks/use_telemetry';
+import { MONITOR_ADD_ROUTE, MONITOR_EDIT_ROUTE } from '../common/constants';
+
+import '@testing-library/jest-dom';
+
+import { PageRouter } from './routes';
+
+describe('PageRouter', () => {
+  beforeEach(() => {
+    jest.spyOn(telemetry, 'useUptimeTelemetry').mockImplementation(() => {});
+  });
+  it.each([MONITOR_ADD_ROUTE, MONITOR_EDIT_ROUTE])(
```

Comment:
Even though this is already merged thought I'd comment on the brilliant use of `it.each`. It's rare to see it in the wild.

Reply:
We have numerous usages of it throughout the `uptime` codebase. It started after @dominiqueclarke joined 🙂.